src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -14,6 +14,11 @@ public static byte[] CloneByteArray(this byte[] src)
             return (byte[])(src.Clone());
         }
 
+        public static KeySizes[] CloneKeySizesArray(this KeySizes[] src)
+        {
+            return (KeySizes[])(src.Clone());
+        }
+
```

Comment:
A valid point but also brings in more ancillary code changes (and inconsistencies with other crypto assemblies) that I don’t want to distract from this change.